pkgs/development/tools/misc/bashdb/default.nix

```diff
@@ -0,0 +1,17 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  name = "bashdb-4.4-0.92";
+
+  src = fetchurl {
+    url =  "mirror://sourceforge/sourceforge/bashdb/bashdb-4.4-0.92.tar.bz2";
```

Comment:
What about using `mirror://sourceforge/bashdb` (instead of `sourceforge/sourceforge`)

Also, consider re-using `name` (or factoring out a `version` attr) in `src.url`.